Add Databricks to the Cloud Data Ingestion documentation (#5556)
braze-inc/braze-docs: _docs/_user_guide/data_and_analytics/cloud_ingestion/integrations.md

- User identifier columns. Your table may contain one or more user identifier columns. Each row should only contain one identifier (either `external_id`, the combination of `alias_name` and `alias_label`, or `braze_id`). A source table may have columns for one, two, or all three identifier types. 
    - `EXTERNAL_ID` - This identifies the user you want to update. This should match the `external_id` value used in Braze. 
    - `ALIAS_NAME` and `ALIAS_LABEL` - These two columns create a user alias object. `alias_name` should be a unique identifier, and `alias_label` specifies the type of alias. Users may have multiple aliases with different labels but only one `alias_name` per `alias_label`.
    - `BRAZE_ID` - The Braze user identifier. This is generated by the Braze SDK, and new users cannot be created using a Braze ID through Cloud Data Ingestion. To create new users, specify an external user ID or user alias. 
- `PAYLOAD` - This is a string or struct of the fields you want to sync to the user in Braze.

#### Step 2: Create a Access Token  

> In order for Braze to access databricks, a personal access token needs to be created.

Reviewer: Databricks

1. In your Databricks workspace, click your Databricks username in the top bar, and then select `User Settings` from the drop down.
2. On the Access tokens tab, click Generate new token.
3. Enter a comment that helps you to identify this token such as Braze CDI, and change the token’s lifetime to no lifetime by leaving the Lifetime (days) box empty (blank).
4. Click Generate.
5. Copy the displayed token, and then click Done.

> Keep the token in a safe place until you need to entry it on the Braze dashboard during integration credintial creation.

Reviewer: ```suggestion
Keep the token in a safe place until you need to enter it on the Braze dashboard during the credential creation step.
```

- User identifier columns. Your table may contain one or more user identifier columns. Each row should only contain one identifier (either `external_id`, the combination of `alias_name` and `alias_label`, or `braze_id`). A source table may have columns for one, two, or all three identifier types. 
    - `EXTERNAL_ID` - This identifies the user you want to update. This should match the `external_id` value used in Braze. 
    - `ALIAS_NAME` and `ALIAS_LABEL` - These two columns create a user alias object. `alias_name` should be a unique identifier, and `alias_label` specifies the type of alias. Users may have multiple aliases with different labels but only one `alias_name` per `alias_label`.
    - `BRAZE_ID` - The Braze user identifier. This is generated by the Braze SDK, and new users cannot be created using a Braze ID through Cloud Data Ingestion. To create new users, specify an external user ID or user alias. 
- `PAYLOAD` - This is a string or struct of the fields you want to sync to the user in Braze.

#### Step 2: Create a Access Token  

In order for Braze to access Databricks, a personal access token needs to be created.

> 1. In your Databricks workspace, click your Databricks username in the top bar, and then select `User Settings` from the drop down.

Reviewer 2: ```suggestion
1. In your Databricks workspace, click your Databricks username in the top bar, and then select **User Settings** from the drop-down.
```

> 2. On the Access tokens tab, click Generate new token.

Reviewer 2: ```suggestion
2. On the Access tokens tab, click **Generate new token**.
```

> 3. Enter a comment that helps you to identify this token such as Braze CDI, and change the token’s lifetime to no lifetime by leaving the Lifetime (days) box empty (blank).

Reviewer 2: ```suggestion
3. Enter a comment that helps you to identify this token, such as "Braze CDI", and change the token’s lifetime to no lifetime by leaving the Lifetime (days) box empty (blank).
```

> 4. Click Generate.

Reviewer 2: ```suggestion
4. Click **Generate**.
```

> 5. Copy the displayed token, and then click Done.

Reviewer 2: ```suggestion
5. Copy the displayed token, and then click **Done**.
```

You will also choose the data type and sync frequency. Frequency can be anywhere from every 15 minutes to once per month. We'll use the time zone configured in your Braze dashboard to schedule the recurring sync. Supported data types are Custom Attributes, Custom Events, Purchase Events, and User Deletes. The data type for a sync cannot be changed after creation. 

{% endtab %}
{% tab Databricks %}

Go to **Partner Integrations** > **Technology Partners**. Find the Databricks page and click **Create new import sync**.

{% alert note %}
If you are using the [older navigation]({{site.baseurl}}/navigation), go to **Technology Partners**.
{% endalert %}

#### Step 1: Add Databricks connection information and source table
Input the information for your Databricks data warehouse and source table, then proceed to the next step.

![]({% image_buster /assets/img/cloud_ingestion/ingestion_16.png %})

#### Step 2: Configure sync details
Next, choose a name for your sync and input contact emails. We'll use this contact information to notify you of any integration errors (e.g., access to the table was removed unexpectedly).

![]({% image_buster /assets/img/cloud_ingestion/ingestion_12.png %})

> You will also choose the data type and sync frequency. Frequency can be anywhere from every 15 minutes to once per month. We'll use the time zone configured in your Braze dashboard to schedule the recurring sync. Supported data types are Custom Attributes, Custom Events, Purchase Events, and User Deletes. The data type for a sync cannot be changed after creation. 

Reviewer 2: ```suggestion
You will also choose the data type and sync frequency. Frequency can be anywhere from every 15 minutes to once per month. We'll use the time zone configured in your Braze dashboard to schedule the recurring sync. Supported data types are custom attributes, custom events, purchase events, and user deletes. The data type for a sync cannot be changed after creation. 
```
